src/pages/home/Home.jsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react'
+import { Link } from 'react-router-dom';
+import './style.css'
 
 const Home = () => {
   return (
-    <div>Home</div>
+    <>
+      <span className="landing-page">Need a loan? Sign up for one now!</span>
+      <div className="signup-btn">
+        <Link to='/signup' className="link">
+          <button>GET YOUR LOAN NOW!</button>
```

Comment:
Button looks good with the CSS. We should probably re-use the `StyledButton `component here.